RecoParticleFlow/PFProducer/python/particleFlow_cff.py

```diff
@@ -15,14 +15,22 @@
 particleFlowTmp.PFEGammaFiltersParameters.allowEEEinPF = cms.bool(False)
 
 # Thresholds for e/gamma PFID DNN 
+# Thresholds for electron: Sig_isolated+Sig_nonIsolated
```

Comment:
might be good to have a reference to an indico presentation here with these numbers

Reply:
Hi @jpata, It should be this one https://indico.cern.ch/event/1096395/#12-egm-updates-for-run-3-prepa., shown at the last PPD meeting by @akapoorcern.

Reply:
right, please put it in the code comment :)

Reply:
ok! :+1: